src/mscorlib/model.xml

```diff
@@ -8152,6 +8152,10 @@
       <Member MemberType="Property" Name="Local" />
       <Member MemberType="Property" Name="Utc" />
     </Type>
+    <Type Name="System.TimeZoneInfo+AdjustmentRule">
+      <!-- CreateAdjustmentRule doesn't have a public contract, but tests and other callers can use Reflection to invoke it, so ensure it is left in mscorlib. -->
```

Comment:
This method was never public in CoreCLR. Using ` Status="ApiFxInternal" ` will restore the method to the state it was before your change.

These tests either never worked, or they use some other loophole to bypass accessibility checks.